gpmp2/kinematics/tests/testArm.cpp

```diff
@@ -296,13 +304,13 @@ TEST(Arm, WAMexample) {
   pvec_exp.push_back(Point3(-0.0060, 0.6534, -0.0582));
   pvec_exp.push_back(Point3(-0.0066, 0.7134, -0.0576));
 
-  vvec_exp.push_back(Vector3(0, 0, 0));
-  vvec_exp.push_back(Vector3(0, 0, 0));
-  vvec_exp.push_back(Vector3(-0.0557, 0.0204, -0.0511));
-  vvec_exp.push_back(Vector3(-0.0606, 0.0234, -0.0595));
-  vvec_exp.push_back(Vector3(-0.0999, -0.0335, -0.0796));
-  vvec_exp.push_back(Vector3(-0.0999, -0.0335, -0.0796));
-  vvec_exp.push_back(Vector3(-0.1029, -0.0333, -0.0976));
+  vvec_exp.push_back(Vector6(0, 0, 0, 0, 0, 0.1));
```

Comment:
Please add a high level comment where you got these numbers from and how to reproduce them.